core-services/ci-secret-generator/_config.yaml

```diff
@@ -88,3 +88,12 @@
   params:
     cluster:
     - app.ci
+- item_name: ci-chat-bot
+  attachments:
+  - cmd: oc --context $(cluster) sa create-kubeconfig --namespace ci $(service_account) | sed "s/$(service_account)/$(cluster)/g"
+    name: sa.$(service_account).$(cluster).config
```

Comment:
Wait, apparently I got this wrong so:

* The chat bot uses the in-cluster config to talk to app.ci (so no need to involve bitwarden there)
* And it has mounted an extra config to talk to api.ci - That one needs to go through bitwarden

Reply:
sorry for the confusion

Reply:
No problem!  Im glad you mentioned it.  We can probably just scrape this PR altogether then and just remove passing the kubeconfig.

Reply:
Sorry for the noise!

Reply:
/close